Wikipedia Citation Templates.js

```diff
@@ -309,6 +316,22 @@ function doExport() {
 				}
 			}
 		}
+
+		// Old publishers may be written as "Last, First" etc. according to
+		// standards, but should be like authors when in citations.
+		// Source: librarian https://it.wikipedia.org/?diff=69781295
+		// "Dozza, Evangelista (1.), eredi"
+		// "Marnef, Jérôme de & Cavellat, Guillaume, veuve"
+		if(item.publisher && properties.date && properties.date < '1831') {
+			// Remove excess space and secondary disambiguations
+			var declutter = item.publisher.replace(/ *\& */g, "; ").replace(/ *\([^)]+\)/g, "");
+			// Shuffle names with disambiguation and remove numbering
+			var shuffle3 = declutter.replace(/ *([^,;]+), +([^,;0-9]+) *[0-9]*, +([^,;]+) */g, "$2 $1 ($3)");
+			// Same, other names
+			var shuffle2 = shuffle3.replace(/ *([^,;]+), +([^,;0-9]+) *[0-9]* */g, "$2 $1");
+			// Use comma list, get rid of space buildup for numbered names
+			properties.publisher = shuffle2.replace(/ *[;] */g, ", ").replace(/  /g, " ");
+		}
```

Comment:
Maybe ZU.trimInternal() would do all the spacing problems. However, I don't think we should do the reorder for all data. If I search for "&" or "," as publisher in my current Zotero library I find for example
 * Hodges, Foster & Co.
 * The Minerals, Metals & Materials Society
 * ProQuest Information & Learning
 * O'Reilly Vlg. Gmbh & Co.
 * Hessisches Ministerium für Landwirtschaft, Forsten und Naturschutz, Landentwicklung
 * Office of Plans and Policy, Federal Communications Commission
 * printed by Henry Hills, printer to the King's most Excellent Majesty, for His Household and Chappel
 * printed for Samuel Smith, at the Princes Arms in St. Paul's Church-yard

The reordering in these example would be wrong. The publication date (<1831) seems for me to be too weak for a condition.